curve_fitting.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        """For loop to avoid repeated the same lines of code with only one word difference"""
</pre><p>docstring must go immediately after the function definition. This won't show up in help().</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/4-21-22-Class/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>